agenta-web/src/lib/helpers/utils.ts

```diff
@@ -74,12 +74,19 @@ export const isAppNameInputValid = (input: string) => {
 type RowType = Record<string, any>
 
 export const convertToCsv = (rows: RowType[], header: (string | undefined)[]): string => {
-    const validHeaders = header.filter((h) => h !== undefined && h in rows[0]) as string[]
-    const headerRow = validHeaders.join(",")
+    const validHeaders = header.filter((h) => h !== undefined && h in rows[0]) as string[];
+    const headerRow = validHeaders.join(",");
+
+    const escapeField = (field: string) => {
+        field = field.trim()
+        return field.includes(",") ? `"${field.replace(/"/g, '""')}"` : field;
```

Comment:
I am a little worried that it can cover every case. Why don't you use a library like https://www.npmjs.com/package/react-csv-downloader or something else?

Reply:
Thanks for comment. 
There are two ways using react-csv-downloader 
1. Using <CsvDownloader> Tag 
  - To this way, we'll need to tag all the place where using this function. a lot of places.
3. Using toCsv function. 
  - Because toCsv method returns a Promise, we would have to modify a lot of code to 'async' in order to this way 

I compare with react-csv-downloader code and updated the code. 
Additional feedback welcome and new PRs also would be appreciated! 
Anyway, I'll feel good if bug was resolved.